tests/test_bridge_domain.rb

```diff
@@ -24,6 +24,11 @@ class TestBridgeDomain < CiscoTestCase
   @@cleaned = false # rubocop:disable Style/ClassVars
 
   def cleanup
+    Vlan.vlans.each do |vlan, obj|
+      # skip reserved vlans
+      next if vlan == '1'
+      obj.destroy
+    end
```

Comment:
This is probably common enough code that it can live in `ciscotest.rb` *a la* 
https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/ciscotest.rb#L188